src/components/Modals.js

```diff
@@ -84,10 +85,12 @@ export const DeleteModal = (props) => {
       open={props.modalOpen}
       onRequestClose={props.onModalClose}
       onRequestSubmit={props.onModalSubmit}
+      additionalText={props.additionalText}
       primaryButtonText="Delete Resource"
       danger
     >
       <span>You are about to delete {name}. This cannot be undone.</span>
+      <p>{props.additionalText}</p>
```

Comment:
need a conditional here so that it's not display if nullOrEmptyString (can use from lazy-z)